tests/safety_replay/helpers.py

```diff
@@ -29,19 +29,19 @@ def is_steering_msg(mode, addr):
 def get_steer_value(mode, to_send):
   torque, angle = 0, 0
   if mode in (Panda.SAFETY_HONDA_NIDEC, Panda.SAFETY_HONDA_BOSCH):
-    torque = to_send.RDLR & 0xFFFF0000
+    torque = ((to_send.data[2] << 8) | to_send.data[3]) << 16
```

Comment:
this doesn't look quite right. it's only 16 bits large, and these are the wrong bytes